cookiecutter/migrate.py

```diff
@@ -64,8 +64,7 @@ def create_dependabot_auto_merge_workflow() -> None:
     steps:
       - name: Auto-merge Dependabot PR
         uses: >-
-          frequenz-floss/dependabot-auto-approve@3cad5f42e79296505473325ac6636be897c8b8a1
-
+          frequenz-floss/dependabot-auto-approve@3cad5f42e79296505473325ac6636be897c8b8a1  # v1.3.2
```

Comment:
Not sure what was wrong, but why not just?

```suggestion
       uses: frequenz-floss/dependabot-auto-approve@3cad5f42e79296505473325ac6636be897c8b8a1  # v1.3.2
```